### PR TITLE
enable RHOBS monitoring for hypershift

### PR DESCRIPTION
**Pull request by rokej**
Signed-off-by: Roke Jung <roke@redhat.com>

# Description of the change(s):
* Enable RHOBS monitoring for hypershift

## Why do we need this PR:
*  

## Issue reference: 
* https://issues.redhat.com/browse/HOSTEDCP-675

## Test API/Unit - Success
```script

```

### CODE REVIEW

File: pkg/install/hypershift.go
Comment (csrwng) [inline pkg/install/hypershift.go:312]: we also need a `ENABLE_RHOBS_MONITORING=1` env var in the job that runs the install. Otherwise, our install fails:
https://github.com/openshift/hypershift/blob/a4fbc63f386561932e07253181d0fce788eb5ba4/cmd/install/install.go#L124-L126
The reason is that during init, we need to know which groupVersion to add to the schema:
https://github.com/openshift/hypershift/blob/a4fbc63f386561932e07253181d0fce788eb5ba4/api/scheme.go#L63-L67

Comment (rokej) [reply pkg/install/hypershift.go:312]: @csrwng Can you please review again?

**Comment by csrwng** [comment]
Thanks @rokej 
/lgtm

**Comment by rokej** [comment]
/hold

should wait for https://github.com/openshift/hypershift/pull/1932 and hypershift operator upgrade before this gets merged

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**
